src/pstl_offload.cpp

```diff
@@ -96,59 +97,311 @@ __original_msize(void* __user_ptr)
     return __orig_msize(__user_ptr);
 }
 
+
+static auto
+__get_original_realloc()
+{
+    using __realloc_func_type = void* (*)(void*, std::size_t);
+
+    static __realloc_func_type __orig_realloc = __realloc_func_type(dlsym(RTLD_NEXT, "realloc"));
+    return __orig_realloc;
+}
+
+inline bool
+__is_ptr_page_aligned(void* __p)
+{
+    return (uintptr_t)__p % __get_memory_page_size() == 0;
+}
+
+struct __hash_aligned_ptr
+{
+    uintptr_t operator()(void* __p) const
```

Comment:
```suggestion
    std::uintptr_t operator()(void* __p) const
```
For all the places where we have `uintptr_t`

Reply:
Done.